VladimirPopov_WebForms/templates/webforms/fields/select_checkbox.phtml

```diff
@@ -0,0 +1,121 @@
+<div>
+    <?php
+    $validate = '';
+    $validate_min = '';
+    $validate_max = '';
+    $options = $this->getFieldOptions();
+    foreach ($this->getFieldOptions() as $i => $option) {
+        $checked = false;
+        $validate_min = '';
+        $validate_max = '';
+        if ($option["checked"]) {
+            $checked = 'checked';
+        }
+        if ($this->getField()->getRequired() && $i == (count($options) - 1)) {
+            $validate = "validate-one-required-by-name";
+            if ($this->getField()->getData('validate_regex')) {
+                $validate .= ' validate-field-' . $this->getField()->getId();
+            }
+            $validate .= ' ' . $this->getFieldClass();
+        }
+        if ($this->getMin() && $i == (count($options) - 1)) {
+            $validate_min = "validate-field-options-min-{$this->getField()->getId()}";
+        }
+        if ($this->getMax() && $i == (count($options) - 1)) {
+            $validate_max = "validate-field-options-max-{$this->getField()->getId()}";
+        }
+        if(!$this->getField()->getValidationAdvice()){
+            $this->getField()->setValidationAdvice(__('Please select one of the options.'));
+        }
+        if (isset($option["value"]) && empty($option['optgroup']) && empty($option['optgroup_close'])) {
+            $label = $option["label"];
+            if ($option["image_src"]) {
+                $label = "<figure><img src='{$option['image_src']}'/><figcaption>{$label}</figcaption></figure>";
+            }
+            ?>
+            <div
+                class='
+                    field
+                    choice
+                    option-<?= $i ?>
+                    checkbox
+                '
+                style='<?= $this->getFieldStyle() ?>'
+            >
+                <input type='checkbox'
+                    name='<?= $this->getFieldName() ?>[]'
+                    id='<?= $this->getFieldId() . $i ?>'
+                    value="<?= htmlspecialchars($option["value"]) ?>"
```

Comment:
use escaping for attributes

- `escapeJs` escaping values in js
- `escapeHtml` text content
- `escapeHtmlAttr` for attributes e.g. `aria-label`
- `escapeUrl` for urls

### examples:
```<?= $block->escapeHtml(__('Update Shopping Cart')) ?>```
```<?= $block->escapeHtmlAttr(__('Clear Shopping Cart')) ?>```
```<?= $block->escapeUrl($block->getUrl('checkout/cart/updatePost')) ?>```